### PR TITLE
Document all the reference shapes and their (current) numbering

### PR DESCRIPTION
**Pull request by fredrikekre**

### CODE REVIEW

**Review by termi-official — CHANGES_REQUESTED** [review]
We could also leave a hint for https://ferrite-fem.github.io/FerriteViz.jl/dev/api/#FerriteViz.elementinfo which displays element information interactively.

File: src/docs.jl [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+Reference triangle, reference dimension 2.
+```
+----------------+--------------------
+Vertex numbers: | Vertex coordinates:
+3               |
+| \\             | v1: 𝛏 = (0.0, 0.0)
+|   \\           | v2: 𝛏 = (1.0, 0.0)
+|     \\         | v3: 𝛏 = (0.0, 1.0)
+1-------2       |
```
[the remark this answers is not in the document]
Comment (fredrikekre) [reply src/docs.jl:34]: Okay, strange.

File: src/docs.jl [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+----------------------------------+----------------------------
+Vertex numbers:                   | Vertex coordinates:
+    4-------/6       4--------6   |
+   /     /   |      /|        |   |  v1: 𝛏 = (0.0, 0.0, 0.0)
+  /   /      |     / |        |   |  v2: 𝛏 = (1.0, 0.0, 0.0)
+ 5 /         |    5  |        |   |  v3: 𝛏 = (0.0, 1.0, 0.0)
+ |          /3    |  1-------/3   |  v4: 𝛏 = (0.0, 0.0, 1.0)
+ |       /        | /     /       |  v5: 𝛏 = (1.0, 0.0, 1.0)
+ |    /           |/   /          |  v6: 𝛏 = (0.0, 1.0, 1.0)
+ 2 /              2 /             |
```
[the remark this answers is not in the document]
Comment (fredrikekre) [reply src/docs.jl:159]: Good suggestion, added!

**Comment by fredrikekre** [comment]
> We could also leave a hint for https://ferrite-fem.github.io/FerriteViz.jl/dev/api/#FerriteViz.elementinfo which displays element information interactively.

Done! :)

**Review by termi-official — APPROVED** [review]
Great and thanks! I went over them once more and I *think* they reflect these docs current implementation.

**Comment by fredrikekre** [comment]
Thanks for the review!

**Pull request merged**